zookeeper-server/src/test/java/org/apache/zookeeper/server/ZooKeeperCriticalThreadMetricsTest.java

```diff
@@ -47,8 +50,8 @@ public void shutdown() {
 
     private class MyPrepRequestProcessor extends PrepRequestProcessor {
 
-        public MyPrepRequestProcessor() {
-            super(new ZooKeeperServer(), new MyRequestProcessor());
+        public MyPrepRequestProcessor() throws IOException {
+            super(new ZooKeeperServer(new FileTxnSnapLog(new File("tmp"), new File("tmp"))), new MyRequestProcessor());
```

Comment:
How does this change relate to the patch?

Reply:
`ZooKeeperCriticalThreadMetricsTest` was failing, and this was my fix to it. Not sure if I should just ignore the error for the PR. What should I do?

Reply:
If the test is failing otherwise, leave it there.